### PR TITLE
import into: switch tikv mode during import

### PR DESCRIPTION
**Pull request by D3Hunter**
### What problem does this PR solve?

Issue Number: ref #42930 

Problem Summary:

### What is changed and how it works?
- add OnTicker to flowHandle to do periodical task, ticker event is triggerred when task is NOT FINISHED and NO ERROR
- keep switch tikv to IMPORT mode during import and switch back on error or finish import(before post process)

### Check List

Tests 

- [ ] Unit test
- [x] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

```release-note
None
```

### CODE REVIEW

**Review by lance6716 — APPROVED** [review]

File: disttask/framework/dispatcher/dispatcher.go [hunk 1]
```diff
@@ -171,6 +171,7 @@ func (d *dispatcher) DispatchTaskLoop() {
 				if d.isRunningGTask(gTask.ID) {
 					continue
 				}
+				// owner changed
```
Comment (GMHDBJD) [inline disttask/framework/dispatcher/dispatcher.go:174]: What does it mean?

**Review by GMHDBJD — APPROVED** [review]
LGTM

File: disttask/framework/dispatcher/dispatcher.go (hunk 1 above)
Comment (D3Hunter) [reply disttask/framework/dispatcher/dispatcher.go:174]: tidb owner changed, so just `setRunningGTask`

**Pull request merged**